resources/models/reservation.py

```diff
@@ -187,7 +189,20 @@ class Reservation(ModifiableModel):
     requested_at = models.DateTimeField(
         null=True, blank=True, verbose_name=_("Requested at")
     )
+
+    invoice_requested = models.BooleanField(
+        default=False,
+        verbose_name=_("Invoice requested by customer"),
+    )
+
+    invoice_requested_at = models.DateTimeField(
+        null=True,
+        blank=True,
+        verbose_name=_("Invoice requested by customer"),
```

Comment:
Should the verbose name be "Invoice requested at"?

Reply:
Maybe "Invoice requested by customer at"